Divers petits ajouts (#862)
Gp2mv3/Syntheses: src/q5/magn-ELEC1350/summary/magn-ELEC1350-summary.tex

\section{Densité de flux électrique, loi de Gauss et divergence}
\subsection{Champ de déplacement électrique}
Dans un matériau \textbf{linéaire}, \textbf{homogène}, \textbf{isotrope} et à réponse \textbf{instantanée}, le champ de déplacement électrique ou densité de flux électrique est donné par
\begin{equation}
	\D = \perm \E.
	\u{\coulomb\per\meter\squared}
	\label{eq:def-d}
\end{equation}

> Pour rappel, $\perm$  représente la permittivité absolue du milieu. Elle décrit la réponse d'un milieu donné à un champ électrique appliqué.\\

Reviewer: ```suggestion
Pour rappel, $\perm$  représente la permittivité absolue du milieu. Elle décrit la réponse d'un milieu donné à un champ électrique appliqué.

```

	\item La norme du champ électrique est donnée par la
	valeur maximum de la dérive du potentiel par rapport à la
	distance ;

> 	\item Le champ électrique se dirige donc d'un potentiel plus élevé vers un plus faible tout en étant  perpendiculaire aux équipotentielles.

Reviewer: Je me demande si les deux phrases ne pourraient pas être gardées, même si elles sont (à première vue) équivalentes.

équipotentielles du matériau et $I$ le courant total
traversant la surface la plus positive du matériau.

\subsection{Effet peau}
\label{subseq:peau}

> Dans un bon conducteur en courant alternatif, les champs EM sont rejetés à la surface, ils ne pénètrent pas au-delà de quelques « profondeurs de peau ». 

Reviewer: ```suggestion
Dans un bon conducteur en courant alternatif, les champs EM sont rejetés à la surface, ils ne pénètrent pas au-delà de quelques \og profondeurs de peau \fg. 
```
Il me semble qu'avec l'insertion des `«»` de manière directe, le principal problème c'est un soucis d'espacement (genre espaces insécables requis alors qu'on a inséré des espaces normales). @Peiffap corrige-moi si je me trompe ou si ça a changé dans LaTeX/Babel french.

\begin{equation}
	\delta = \sqrt{\frac{2}{\omega \mu \sigma}}
\end{equation}

> Le coefficient $\delta$ représente la profondeur de peau, c’est-à-dire la profondeur à laquelle le champ a diminué de $\frac{1}{e}$. A 5 profondeurs de peau, le champ est réduit à 1\% du champ à la surface du métal. Par exemple pour le cuivre, $\delta$ vaut 9 mm à 50 Hz,7 $\mu$m à 100 MHz et 0,7 $\mu$m à 10 GHz ! 
> Voilà pourquoi on met 3 fils au lieu d’un seul plus gros pour le transport d’électricité !

Reviewer: ```suggestion
Le coefficient $\delta$ représente la profondeur de peau, c’est-à-dire la profondeur à laquelle le champ a diminué de $\frac{1}{e}$. A 5 profondeurs de peau, le champ est réduit à 1\% du champ à la surface du métal. Par exemple pour le cuivre, $\delta$ vaut \SI{9}{mm} à \SI{50}{Hz}, \SI{7}{\micro\meter} à \SI{100}{MHz} et \SI{0.7}{\micro\meter} à \SI{10}{GHz}! 
Voilà pourquoi on met 3 fils au lieu d’un seul plus gros pour le transport d’électricité!
```

TODO: ondes planes, vecteur d'onde, vecteur de Poynting, etc.

Des charges électriques accélérées sont responsables de l’apparition d’une onde électromagnétique.

\part{Lignes de transmission}

Une ligne de transmission est un conducteur électrique le long duquel un signal courant-tension
se propage. En pratique, on utilise le formalisme de ligne de transmission quand la fréquence est
grande (càd la longueur d'onde est petite ou du même ordre de grandeur que la taille du conducteur).
À basse fréquence, il n'y a pas besoin d'utiliser le formalisme de ligne de transmission car on peut
considérer le conducteur comme un conducteur parfait (courant et tension identiques aux deux
extrémités, éventuellement avec résistance série), ce qu'on fait souvent en théorie des circuits. Mais lorsque $\vartheta(l)>\vartheta(\lambda)$, les lignes de transmission deviennent des structures qui propagent des EM, elles ne peuvent plus être représentées par des éléments de circuits localisés ($R, L, C$ discrets). Cette structure est désormais le siège d’une onde de tension  $V(z,t) = f_V(\omega t \pm \beta z) \neq V(t)$ et d’une
onde de courant  $I(z,t) = f_I(\omega t \pm \beta z) \neq I(t)$.

> Ex :  lorsqu’un signal sinusoïdal est envoyé sur une ligne de transmission, il subira un délai.\\

Reviewer: ```suggestion

Ex:  lorsqu’un signal sinusoïdal est envoyé sur une ligne de transmission, il subira un délai.

```
Btw, les retours à la ligne sont pas trop cohérents (5 lignes de cut à 80 puis une ligne de >350 caractères puis un cut et une ligne de < 80 caractères) : peut-être uniformiser vers des cuts à 80 sur tout le paragraphe?

\begin{figure}[ht]
	\centering
	\includegraphics[scale=0.5]{img/telegraph-circuit.png}

> 	\caption{Schéma d'une ligne de transmission. Les éléments du circuits sont exprimés par unité de longueur $dz$.}

Reviewer: ```suggestion
	\caption{Schéma d'une ligne de transmission. Les éléments du circuits sont exprimés par unité de longueur $\dif{z}$.}
```

	\fdif{I_s}{z} = -(G+j\omega C)V_s
\end{equation}

On dérive pour obtenir l'équation des télégraphistes:

\begin{align}

> 	\fdif{^2V_s}{z^2} = ZYV_s \\
> 	\fdif{^2I_s}{z^2} = ZYI_s

Reviewer: ```suggestion
	\fdif{^2V_s}{z^2} &= ZYV_s \\
	\fdif{^2I_s}{z^2} &= ZYI_s
```
D'ailleurs, est-ce que c'est voulu les numéros d'équation ?

\subsection{Quelques grandeurs caractéristiques}

\begin{mydef}

> Le coefficient de réflexion en $z$ se définit, dans le cas sans pertes, comme\footnote{On retrouve bien le coefficient de Fresnel pour un incidence normale.}:

Reviewer: ```suggestion
Le coefficient de réflexion en $z$ se définit, dans le cas sans pertes, comme\footnote{On retrouve bien le coefficient de Fresnel pour une incidence normale.}:
```

\section{Transitoires}

En régime, la charge est connue du générateur. En transitoire, le générateur ne connait pas la charge jusqu’au moment où l’onde reviendra jusqu’a lui. Càd pas avant un temps égal à $\frac{2l}{c}$ où $l$ est la longueur de la \emph{première} ligne.

> Donc le générateur "suppose" que la ligne présente une résistance $R_c$ en tout point au front d’onde.\\

Reviewer: ```suggestion
Donc le générateur ``suppose'' que la ligne présente une résistance $R_c$ en tout point au front d’onde.
```

	Z_c i(z, t) &= f_+(t-z/c) - f_-(t+z/c) \\
\end{align*}

> avec \[Z_c \stackrel{\Delta}{=} \frac{V_0}{I_0}= \sqrt{\frac{L}{C}}\]

Reviewer: On a `\eqdef` pour ça dans `eplcommon.sty` :)
